Section 3 - Terraform Provisioners/local-exec.tf

```diff
@@ -1,8 +1,37 @@
 resource "aws_instance" "myec2" {
-   ami = "ami-082b5a644766e0e6f"
-   instance_type = "t2.micro"
+  ami           = "ami-082b5a644766e0e6f"
```

Comment:
**Tenable.cs remediation**
 
**[MEDIUM]: Ensure virtual private cloud (VPC) is configured for AWS EC2 instances**
 
**Vulnerability**
By default AWS EC2 instances are launched in default VPC which is not recommended for production grade. AWS VPCs provides the controls to facilitate a formal process for approving and testing all network connections and changes to the firewall and router configurations.
 
**Remediation**
Amazon Virtual Private Cloud (Amazon VPC) enables you to define a virtual network in your own logically isolated area within the AWS cloud, known as a virtual private cloud or VPC. You can create AWS resources, such as Amazon EC2 instances, into the subnets of your VPC. Your VPC closely resembles a traditional network that you might operate in your own data center, with the benefits of using scalable infrastructure from AWS.

source: tenablecs-